test/model/transaction/MosaicSupplyRevocationTransaction.spec.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2018 NEM
```

Comment:
2021 instead?

Reply:
Updated, we may need to revisit the whole licensing for the new foundation